gcp/api/server.py

```diff
@@ -100,7 +108,20 @@ def QueryAffected(self, request, context: grpc.ServicerContext):
 
     version.
     """
-    results, next_page_token = do_query(request.query, context).result()
+    query_context = QueryContext(
+        service_context=context,
+        request_start_time=datetime.now(),
+        page_token=None,
+        total_responses=ResponsesCount(0))
+
+    try:
+      results, next_page_token = do_query(request.query, query_context).result()
```

Comment:
Can we add a log as discussed to log the fact that we paginated?

Reply:
(also need a similar log fot the batch case)

Reply:
Yep, done